src/mesh/voxel_cloud.rs

```diff
@@ -520,4 +520,34 @@ mod tests {
 
         insta::assert_json_snapshot!("sphere_after_voxelization", &voxel_cloud);
     }
+
+    #[test]
+    fn test_voxel_cloud_single_voxel() {
+        let mut voxel_cloud = VoxelCloud::new(
+            &Point3::origin(),
+            &Vector3::new(1, 1, 1),
+            &Vector3::new(1.0, 1.0, 1.0),
+        );
+        voxel_cloud.set_voxel_at_relative_coords(&Point3::new(0, 0, 0), true);
+
+        insta::assert_json_snapshot!("single_voxel", &voxel_cloud);
+    }
+
+    #[test]
+    fn test_voxel_cloud_single_voxel_to_mesh() {
```

Comment:
Maybe append `_produces_synchronized_mesh`, if that's really what this is testing.